implants/lib/eldritch/src/time/now_impl.rs

```diff
@@ -0,0 +1,19 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+use anyhow::Result;
+
+pub fn now() -> Result<u64> {
+    Ok(SystemTime::now().duration_since(UNIX_EPOCH)?.as_secs())
```

Comment:
Hell yeah UNIX EPOCH 😎

Is there a reason (for windows folks) to do Windows EPOCH on windows hosts?

Reply:
I don't think so, tbh I need to make a timestamp to epoch eldritch function and vice versa to make this easier for ppl.